lib/compat/wordpress-6.5/class-wp-navigation-block-renderer.php

```diff
@@ -79,6 +79,23 @@ private static function should_load_view_script( $attributes, $inner_blocks ) {
 		return ( $has_submenus && ( $attributes['openSubmenusOnClick'] || $attributes['showSubmenuIcon'] ) ) || $is_responsive_menu;
 	}
 
+	/**
+	 * Returns the markup for a single inner block.
+	 *
+	 * @param WP_Block $inner_block The inner block.
+	 * @return string Returns the markup for a single inner block.
+	 */
+	private static function get_markup_for_inner_block( $inner_block ) {
+		$inner_block_content = $inner_block->render();
+		if ( ! empty( $inner_block_content ) ) {
+			if ( in_array( $inner_block->name, static::$needs_list_item_wrapper, true ) ) {
```

Comment:
Can we extract the conditional so the concept of "block needs a wrapper" is stored in a well renamed variable.

That means I don't have to study the conditional to understand the code.

Reply:
done in 295bd0370c